### ISSUE TITLE
BUG: Google Login Fail for Reauthenticating Access Token

### ISSUE THREAD

**Issue by Paladz** [finndev/PokeBuddy#39]
PC: Windows 7 
.NET Version: 4.5.2
Login Method: Google mail + Password

After Pokebuddy run for 30 mins, I guess the token is expired so the hack is try to re-login for get the new token. But for some how the login fail  then it frozen the app.
The funny thing is if I stop the app then click the start again, everything work.
Attached the log below:

[P - 01:32:57] [UI] Updating...
[P - 01:32:57] [!] Access token expired. Reauthenticating...
[P - 01:32:57] Doing Google login...

**Comment by brijei** [comment]
two possibility's: 1. Google token 2. pogo servers are down. I keep getting updates pogo servers are up and down multiple times these 3 hours.

**Comment by Paladz** [comment]
@brijei 
Thanks for response this issue
I've test this bug for 7 times. Every-time it fails on exactly 30 mins. So I don't think it's  pogo servers issue. Actually once I open 2 Pokebuddy, One app reached 30 mins then the bug happen, but the another one is keep  running

**Comment by brijei** [comment]
Google is a pain in the ass on my phone it's keep forcing me to relogin really annoying. the app will reload every 30 min that's normal.

**Comment by Paladz** [comment]
I don't think this bug is on either google nor pogo side, should be on the pokebuddy side. But still, Pokebuddy is awesome

**Comment by brijei** [comment]
the Google login method could be better yes. but remember it's still a beta.
